### DISCUSSION TITLE
InputGroup InputNumber display bug

### DISCUSSION THREAD

**Discussion by jnlinn** [primefaces/community#2434] [PrimeReact] [answered]
Hello, in the newly released v10.8.0 I found an issue with InputGroup InputNumber width. Please see the image below from demo page:

![2024-08-01 10_21_41-React InputGroup Component — Mozilla Firefox](https://github.com/user-attachments/assets/b3edf203-e2c0-468c-a138-c1d2aa6fe202)

**Comment by melloware** [comment, marked as answer]
Looks like it was reported here: https://github.com/primefaces/primereact/issues/6962
